tf2onnx/tfonnx.py

```diff
@@ -1747,6 +1750,56 @@ def sparse_softmax_cross_entropy_with_logits_op(ctx, node, name, args):
     return [onehot, log_softmax, mul1, reduce_sum, mul2, res]
 
 
+def floordiv_op(ctx, node, name, args):
+    # T output = floordiv(T x, T y)
+    res = [node]
+    node.type = "Div"
+    dtype = ctx.get_dtype(node.input[0])
+    if dtype in [onnx_pb.TensorProto.FLOAT, onnx_pb.TensorProto.FLOAT16, onnx_pb.TensorProto.DOUBLE]:
+        new_node_name = utils.make_name("floor_div_res")
+        floor_res = ctx.insert_new_node_on_output(op_type="Floor", output_name=node.output[0],
+                                                  name=new_node_name)
+        ctx.copy_dtype(node.output[0], floor_res.output[0])
+        ctx.copy_shape(node.output[0], floor_res.output[0])
+        res.append(floor_res)
+    return res
+
+
+def floormod_op(ctx, node, name, args):
+    # T output = floormod(T x, T y)
```

Comment:
use the real op name FloorMod instead of floormod

Reply:
Intuitively, i would use "FloorMod"

while legacy code forces me to use "floormod", such as gathernd_op,  minmax_op, placeholder_op, expanddims_op7....